.github/actions/install-ubuntu-default-tools/action.yml

```
@@ -0,0 +1,30 @@
+name: "Install Ubuntu default tools"
+description: "Install common Ubuntu tools (e.g.: docker, git, zip, unzip, python3)"
+inputs:
+  package-names-list:
+    description: "The list of package names to be installed separated by a whitespace"
+    required: false
+runs:
+  using: composite
+  steps:
+    - name: "Install required tools"
```

```suggestion
    - name: "Install common tools"
```

May or may not be required by the consumer, so "common" seems more honest.

---

Done.